src/android/java/io/jxcore/node/JXcoreExtension.java

```diff
@@ -4,37 +4,30 @@
 
```

Comment:
This file could really use some re-factoring when it comes to if statements. There are several examples where you have big if statements followed by a tiny else (usually an error). Just checking for the error upfront and then de-indenting the main if would make for more readable code.